AppManifest.json

```diff
@@ -34,10 +34,10 @@
             ],
             "dapr": {
```

Comment:
What about those manifests:
* https://github.com/eclipse-velocitas/vehicle-app-cpp-sdk/blob/main/examples/seat-adjuster/AppManifest.json
* https://github.com/eclipse-velocitas/vehicle-app-cpp-sdk/blob/main/examples/set-data-points/AppManifest.json

Shall we align them?

Reply:
done